examples/readme.md

```diff
@@ -37,7 +37,7 @@ Full documentation and additional training examples are available on [http://www
 
 ## Installation
 
-- [Sign up (free) on comet.com and grab your API key](https://www.comet.com/signup)
+- [Sign up (free) on comet.com and grab your API key](https://www.comet.com/signup/?framework=llm)
```

Comment:
Right now we have PACKAGE_README file used in PyPi, please duplicate these changes to it as well

Reply:
@alexkuzmik Added framework=llm to the only signup link on the the PACKAGE_README md file

Just to confirm: `comet-ml/comet-llm/examples/readme.md` is different than `comet-ml/comet-llm/readme.md`. Let me know if if there's anything else you'd like me to change to any of these READMEs!